src/pages/governance/_components/ProposalCard.tsx

```diff
@@ -23,14 +23,19 @@ import { useCycleEndDate } from "../shared/useCycleEndTime";
 import { OnChainGovernanceTitles } from "../enum/onChainGovernanceTitles";
 import { getSecondsPerBlock } from "../shared/getSecondsPerBlock";
 import { EmergencyChip } from "./EmergencyChip";
+import { VotingResultMobile } from "./VotingResult";
+import { ProposalsVotes } from "../index.page";
+import { VoteCount } from "../shared/getVoteCount";
 
 export function ProposalCards({
   proposals,
+  proposalsVotes,
   currentBlockHeight,
   currentBlockMedianTime,
   userQueryProposalStatus,
 }: {
   proposals: GovernanceProposal[];
```

Comment:
A `proposal` is of the `GovernanceProposal` type which is fixed (defined in whale api repo).

For that reason, proposals votes is its own prop in the format of an object, to allow performant subscripting. 'Smuggling' a proposal's votes into `queryProposal` is not possible without overwriting the type declaration via typecasting.